networkx/algorithms/centrality/current_flow_betweenness.py

```diff
@@ -122,14 +122,14 @@ def approximate_current_flow_betweenness_centrality(
         msg = f"Number random pairs k>kmax ({k}>{kmax}) "
         raise nx.NetworkXError(msg, "Increase kmax or epsilon")
     cstar2k = cstar / (2 * k)
-    for i in range(k):
+    for _ in range(k):
         s, t = seed.sample(range(n), 2)
         b = np.zeros(n, dtype=dtype)
         b[s] = 1
         b[t] = -1
         p = C.solve(b)
         for v in H:
-            if v == s or v == t:
+            if v in (s, t):
```

Comment:
This might be clearer with naming the pair `(s, t)` earlier. e.g.
```python
s, t = pair = seed.sample(range(n), 2)
...
...
    if v in pair:
```